src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -240,11 +240,17 @@ class Wedge3D {
    * somewhere in between
    * \param with_equiangular_map Determines whether to apply a tangent function
    * mapping to the logical coordinates (for `true`) or not (for `false`).
-   * \param halves_to_use Determines whether to use the logical xi
-   * coordinates in the [0,1] interval (value of `UpperOnly`) of the full wedge,
-   * the coordinates in the [-1,0] interval (value of `LowerOnly`) of the
-   * full wedge, or the full wedge entirely (value of `Both`). Half wedges are
-   * currently only useful in constructing domains for binary systems.
+   * \param halves_to_use Determines whether to split the Wedge3D in half. If
```

Comment:
How about "split the Wedge3D in half" -> "construct a full wedge or only half a wedge."

And similarly below in two places "If the wedge is split in half" -> "If constructing only half a wedge".

(reasoning: "split a wedge in half" sounds like you are constructing two halves, but this class constructs only one half).